docs/guide/tutorials.rst

```diff
@@ -0,0 +1,244 @@
+	``metaValue_baseline_delta_2_height_l,metaValue_baseline_delta_2_height_u,``
+	``metaValue_points_across_baseline_l,metaValue_points_across_baseline_u,``
+	``metaValue_points_across_half_height_l,metaValue_points_across_half_height_u,``
+	``metaValue_logSN_l,``
+	``metaValue_logSN_u,intensity_l,intensity_u,overall_quality_l,overall_quality_u``.
+
+	A sample row may be written as follows : 
+	``Serotonin,0,5.00E+06,1,1.00E+06,0.1,1,0.001,0.25,0,2,0.8,2.5,``
+	``-0.25,0.25,20,500,5,500,1,1.00E+06,0,1.00E+15,0,1.00E+12``.
+
+#. features
+	This folder contains features info in the 
+	`featureXML <https://raw.githubusercontent.com/OpenMS/OpenMS/develop/share/OpenMS/SCHEMAS/FeatureXML_1_9.xsd>`_ format.
+
+#. mzML
+	This folder contains `mzML <https://www.psidev.info/mzML>`_ files which descripes raw spectrometer value, 
+	a single mzML file usually encapsulates all the
+	information extracted from a single MS run.
+
+#. parameters.csv
+	This file contains a list of parameters for the RawDataProcessor functions, 
+	the values stored are organised by the header :
+	``function,name,type,value,default,restrictions,description,``
+	``used_,comment_,comparator``, a sample entry may look like this :
+	``MRMMapping,precursor_tolerance,float,``
+	``0.0009,0.1,0.1,Precursor tolerance when mapping (in Th),TRUE,,``.
+
+#. quantitationMethods.csv
+	This file contains information about various quantitation methods and their values. 
+	The header is standarised as follows
+	``IS_name,component_name,feature_name,concentration_units,llod,ulod,lloq,uloq,``
+	``correlation_coefficient,actual_concentration,n_points,transformation_model,``
+	``transformation_model_param_slope,transformation_model_param_intercept,``
+	``transformation_model_param_x_weight,transformation_model_param_y_weight,``
+	``transformation_model_param_x_datum_min,transformation_model_param_x_datum_max,``
+	``transformation_model_param_y_datum_min,transformation_model_param_y_datum_max``.
+	A sample entry in the file may looks like this :
+	``s7p.s7p_1.Heavy,s7p.s7p_1.Light,peak_apex_int,uM,,,,,,,,``
+	``linear,1.88269,1.93E-04,,,1.00E-15,1.00E+15,1.00E-15,1.00E+15``
+
+#. sequence.csv
+	A list of injections are store in this file following the header schema :
+	``sample_name,sample_group_name,sequence_segment_name,sample_type,``
+	``original_filename,batch_name,rack_number,plate_number,pos_number,``
+	``inj_number,dilution_factor,inj_volume,inj_volume_units,operator_name,``
+	``acq_method_name,proc_method_name,acquisition_date_and_time,scan_polarity,``
+	``scan_mass_low,scan_mass_high``.
+	A sample entry may look like the following :
+	``170808_Jonathan_yeast_Sacc2_1x,group1,sequence1,Unknown,``
+	``170808_Jonathan_yeast_Sacc2_1x,BatchName,,,,2,,3,uL,,MethodName,,,,,``.
+
+#. standardsConcentrations.csv
+	This file contains a list of sample names and component names along with 
+	concentration values and dilution factors. The header has
+	the following format : 
```

Comment:
I would also add that "This file is required for automated calibration curve fitting".